src/layer/AbstractSentinelHubV1OrV2Layer.ts

```diff
@@ -131,7 +131,7 @@ export class AbstractSentinelHubV1OrV2Layer extends AbstractLayer {
       ...this.getFindDatesAdditionalParameters(),
```

Comment:
Should `getFindDatesAdditionalParameters()` also be renamed (to `getFindDatesUTCAdditionalParameters` I suppose)?

Reply:
True, thank you, fixed.

Reply:
Nice, resolving.